assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -256,16 +256,18 @@ const useQuickActions = () => {
         label: ACTION_TEXT.CLEAR_ANIMATIONS,
         onClick: () => handleClearAnimations(selectedElement?.id),
         separator: 'top',
+        disabled: !selectedElementAnimations?.length,
```

Comment:
Quick question: will this actually make the button _disabled_, or do we use `aria-disabled`? `disabled` will hide it completely from screen readers, which is not good. So if that's the case, this should be addressed.

Reply:
Yeah it does make the button disabled, but I can change this. These props are passed directly to the button so it would be easy to replace that with aria-disabled.

Reply:
@swissspidy chatted with some friends who are familiar with accessibility. They said that the intended behavior is for screen readers to skip this field. Just to double check: do we want to use `aria-disabled` here instead of `disabled`?

https://stackoverflow.com/a/38059790

Reply:
Since this is of course out of scope for this PR (I was just curious), I suggest opening a ticket to discuss this so we can define expected behavior even beyond Quick Actions (and get some in-house a11y feedback to be 100% certain)

Reply:
this is a tricky one, isn't it 🤔  

I think this is absolutely something we should figure out what we want to do consistently in the app. Looking at other disabled buttons, the redo and undo buttons are an easy one since they are both disabled on a new story - those are skipped over by the keyboard since they don't do anything. 

The buttons in these menus are extra tricky, to me, because it's also a _menu_ so sure, they could be menu items that we acknowledge as disabled and let them be announced. 

Another thought for these menus specifically, this was samuel's thought i'm just mentioning it, to not show options in these menus if they aren't pertinent - kind of a clever way to skirt the `disabled` quandary!